controllers/eventController.js

```diff
@@ -0,0 +1,12 @@
+var mongoose = require('mongoose');
+var Event = require('../models/Event').Event;
+var request = require('request');
+var TOKEN = 'Bearer IHQURKU7V75JVWPZ2OXN';
```

Comment:
Yeah I just stuck it there b/c I didn't know how to handle configs like that for a node app. It's the token for the `admin@coachkalani.com` account, which apparently is the only one with access to Developer settings on Eventbrite.

Can we file an issue and put that in subsequent PR?